src/corg/working-on-nuxeo-sources/unit-testing.md

```diff
@@ -730,7 +713,7 @@ None
 
 #### Usage
 
-```java
+```
```

Comment:
Keep the Java formatting

Reply:
This messed up the formatting in my local environment, but ok.